guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
<p>Please rename the file to how-retinanet-works,ipynb to be consistent with the other guides in this folder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Half of the files in deep learning directory are named using underscores and other half using dashes. What's the norm, in that case?

Reply:
Half of the files in deep learning directory are named using underscores and other half using dashes. What's the norm, in that case?